src/components/directory-content.tsx

```diff
@@ -329,14 +325,15 @@ const DirectoryContent = () => {
     );
 
     const getLink = useCallback(
```

Comment:
To resolve the Sonar Reliability issue:
```ts
    const getLink = useCallback(
        (elementUuid: string, objectType: string): string | undefined => {
            const app = appsAndUrls?.find(isStyduMetadata);
            const resource = app?.resources?.find((res) => res.types.includes(objectType));
            return app && resource && app.url + resource.path.replace('{elementUuid}', elementUuid);
        },
        [appsAndUrls]
    );
```

with outside of `DirectoryContent` component
```ts
function isStyduMetadata(appMetadata: CommonMetadata): appMetadata is StudyMetadata {
    return appMetadata.name === 'Study';
}
```

Reply:
@TheMaskedTurtle said to ignore the sonar issue and do the fix in another PR.